doc/layers.md

```diff
@@ -1,12 +1,18 @@
 # Layers
 
-Layers are an useful feature that lets you group renderization by context, as well as allow you to pre-render things. That enables, for example, the renderization in memory of parts of your game that don't change much, like a background, and by doing that, freeing resources for more dynamic content that needs to be rendered every loop cycle.
+Layers are an useful feature that lets you group rendering by context, as well as allow you to
+pre-render things. This enables, for example, rendering parts of your game that don't change much in
+memory, like a background. By doing this, you'll free resources for more dynamic content that needs
```

Comment:
`free resources` is a bit unclear here I think as you are basically replacing cpu resources with memory resources (like any cache). I think `free processing time` will cause less confusion as resource can lead people to think memory and that is the opposite of what this is for